packages/react/src/components/HorizontalRule/HorizontalRule.js

```diff
@@ -0,0 +1,53 @@
+/**
+ * Copyright IBM Corp. 2016, 2018
+ *
+ * This source code is licensed under the Apache-2.0 license found in the
+ * LICENSE file in the root directory of this source tree.
+ */
+
+import PropTypes from 'prop-types';
+import React from 'react';
+import classnames from 'classnames';
+import settings from 'carbon-components/es/globals/js/settings';
+
+import '@ibmdotcom/styles/scss/components/horizontalrule/_horizontalrule.scss';
+
+const { prefix } = settings;
+/**
+ * Horizontal Rule component
+ *
+ * @param {object} props props object
+ * @param {string} props.type type of rule (solid or dashed)
+ * @param {string} props.color color of the rule
```

Comment:
@annawen1 minor thing, might help to include the available options in the jsdocs under the param descriptions (can probably include all of the options you have for the knobs)

Reply:
yep done